release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3762,4 +3762,35 @@ If these conditions are not met, the pod can fail at admission with `UnexpectedA
 [id="ocp-4-12-21-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+
+
```

Comment:
This seems  like a lot more empty lines than we have done in the past?

Reply:
Well spotted. Removed these extra lines. Used the empty lines as markers for the last minute additions. All fixed now. Thanks!